crates/parcel_filesystem/Cargo.toml

```diff
@@ -7,6 +7,8 @@ description = "FileSystem wrapper trait for use in Parcel codebase."
 [dependencies]
 dashmap = "5.5.3"
 mockall = "0.12.1"
+gxhash = "3.4.1"
```

Comment:
Last I checked this won't compile on musl and some other platforms

Reply:
Changing to xxhash doesn't change the results very much so I think I'll replace with that.